data/human/human missions.txt

```diff
@@ -1961,6 +1961,7 @@ mission "Terraforming 1"
 	passengers 2
 	
 	to offer
+		"reputation: Republic" >= 0
```

Comment:
```suggestion
		"reputation: Republic" < 0
```

You could do that, just to save a character ;)

Reply:
I think the point is that you can't get the mission if the Republic wants to shoot you. (You also used the less than symbol instead of the greater than)

Reply:
Yeah sorry I'm stupid, I forgot that positive values mean they're mad at you.

Reply:
I'd like to get a bit more input about whether they should accept someone at 0 rep before I commit your suggestion.

Reply:
For me they should, why wouldn't they accept someone at 0 reputation? 0 means they're friendly.

Reply:
I thought your suggestion was about making it so they wouldn't accept someone at 0 rep.

Reply:
> Yeah sorry I'm stupid, I forgot that positive values mean they're mad at you. It should then be greater and not greater or equal, since 0 is just neutral: default reputations are always at 0.

No, you had it right to begin with. But those are the conditions to make it offer, rather than conditions which block it from offering, so `> 0` only works if you have better than default rep, and the original `>= 0` works with default rep as well.

I think the original is fine as-is here.

Reply:
Yeah sorry, I don't know what was in my mind that time. My brain just messed up.